### PR TITLE
vtk, vtk6: consolidate patches, remove unneeded changes, fix cmake error

### LINKED ISSUE #1087: vtk fails to build in a directory with ++
Similar to #1030 (using CMake's `MATCHES` against a filepath).

Build log: https://gist.github.com/starius/b1b6690c2b92b7ff63dd

### PR DESCRIPTION
**Pull request by starius**
vtk:

  * https://github.com/mxe/mxe/commit/2ffb7835cd8ea163ee4dadf11784b6d893631ea8 consolidate patches
  *  57d448048d2faa5dd64b17e9948fc2f3b58a0fde remove unneeded changes (removing trailing spaces)
  * https://github.com/mxe/mxe/commit/b5dcff579d68ddf42370ef8732c8aa6972352e6b  fix #1087

vtk6:

  * https://github.com/mxe/mxe/commit/e5273585a16412e94381359dcad003dc2bfdaa82 consolidate patches

### CODE REVIEW

File: src/vtk-1-fixes.patch [hunk 1]
```diff
@@ -0,0 +1,538 @@
+This file is part of MXE.
+See index.html for further information.
+
+Contains ad hoc patches for cross building.
+
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Muellni <mmuellen@gmx.de>
+Date: Tue, 25 Oct 2011 23:32:42 +0200
+Subject: [PATCH] add file TryRunResults.cmake
```
Comment (tonytheodore) [inline src/vtk-1-fixes.patch]: I wanted to try replacing these using the prefixed cmake wrapper and the [common `TryRunResults.cmake`](https://github.com/mxe/mxe/blob/master/src/cmake/modules/TryRunResults.cmake)

File: src/vtk6-1-fixes.patch [hunk 2]
```diff
@@ -0,0 +1,86 @@
+This file is part of MXE.
+See index.html for further information.
+
+Contains ad hoc patches for cross building.
+
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Martin=20M=C3=BCllenhaupt?= <muellni@netlair.de>
```
Comment (tonytheodore) [inline src/vtk6-1-fixes.patch]: What happened there? Is it something `git` does automatically?

Comment (starius) [inline src/vtk6-1-fixes.patch]: Git escaped Unicode characters in the author name (Martin Müllenhaupt <muellni@netlair.de>).

File: src/vtk-1-fixes.patch (hunk 1 above)
Comment (starius) [inline src/vtk-1-fixes.patch]: I think, switching to prefixed cmake wrapper and the common TryRunResults.cmake should go to other pull request.

**Comment by tonytheodore** [comment]
Thanks!

**Pull request merged**